esi_leap/tests/db/sqlalchemy/test_api.py

```diff
@@ -234,12 +234,21 @@ def test_offer_verify_contract_availability(self):
         end = now + datetime.timedelta(days=87)
         api.offer_verify_contract_availability(offer, start, end)
 
+    def test_offer_get_conflict_times(self):
+        o1 = api.offer_create(test_offer_1)
+        self.assertEqual(api.offer_get_conflict_times(o1), [])
+
+    def test_offer_get_first_availability(self):
+        o1 = api.offer_create(test_offer_1)
+        self.assertEqual(api.offer_get_first_availability
+                         (o1.uuid, o1.start_time,), None)
+
```

Comment:
I think I'd update these tests so that the tested functions return something beyond empty results. Just as a quick refresher: an offer is created with a start and end time; by default its availability is that start time and end time. Now let's say someone creates a contract from c.start to c.end:

START----------------------c.start----------c.end----------------------END

The new availabilities should be START->c.start and c.end->END.

I assume a conflict time would be the inverse - so c.start-->c.end.